internal/pkg/util/starter/starter.go

```diff
@@ -106,6 +106,9 @@ func Run(name string, config *config.Common, ops ...CommandOp) error {
 		return fmt.Errorf("while initializing starter command: %s", err)
 	}
 
+	sylog.Debugf("Setting GOGC=off for starter")
```

Comment:
We can remove those two lines, this is not handled by our starter

Reply:
I removed `GOGC` lines